mount/losetup_linux.go

```diff
@@ -181,7 +189,12 @@ func removeLoop(loopdev string) error {
 
 // Attach a specified backing file to a loop device
 func AttachLoopDevice(backingFile string) (string, error) {
```

Comment:
Are the devmapper tests which use this still going to run into issues using the loop device?

Reply:
Since LoopParams doesn't have Autoclear here. So closing the file won't destruct the loopback device.

Reply:
OK, I guess we can separate that some. I thought we were still seeing some related issues on devmapper. In this case, could/should it just return the file with autoclear to avoid detach? If there are no leaks there though that might not help the running out of loop devices issues there.